src/java/org/apache/cassandra/db/guardrails/CustomGuardrailsFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.db.guardrails;
+
+public abstract class CustomGuardrailsFactory
+{
+    public static GuardrailsFactory make(String customImpl)
+    {
+        try
+        {
+            return (GuardrailsFactory) Class.forName(customImpl).newInstance();
```

Comment:
I think we have some tooling around this stuff in FBUtilities or so ... instantiation of custom classes with all bells and whistles.

Reply:
`org.apache.cassandra.utils.FBUtilities#construct(java.lang.String, java.lang.String)` does the same

Reply:
Good idea, I have changed the method to use `FBUtilities#construct` and moved it to `GuardrailsFactory`